packages/api-core/src/graphql/schema.graphql

```diff
@@ -34,6 +34,130 @@ enum MassUnit {
   oz
 }
 
+"Logical Operator Types for filtering"
+enum LogOpTypes{
+  "AND"
+  AND
+
+  "OR"
+  OR
+}
+
+"Relational Operator Types for filtering"
+enum RelOpTypes{
+  "Begins With used with String types"
+  beginsWith
+
+  "Ends With used with String types"
+  endsWith
+
+  "Equal"
+  eq
+
+  "Greater Than"
+  gt
+
+  "Greater Than or Equal"
+  gte
+
+  "In"
+  in
+
+  "Less Than"
+  lt
+
+  "Less Than or Equal"
+  lte
+
+  "Not Equal"
+  ne
+
+  "Not In"
+  nin
+
+  "Regex"
+  regex
+}
+
+"Flag to specify the number of levels used in Filter Search"
+enum FilterLevels {
+  "Use Filter with 1 level"
+  ONE
+
+  "Use Filter with 3 levels"
+  THREE
+
+  "Use Filter with 2 levels"
+  TWO
+}
+
+"Single Condition for filterSearch"
+input SingleConditionInput {
+  "Value to filter if it is Boolean input"
+  boolValue: Boolean
+
+  "Flag to set if the regex is case insensitive"
+	caseSensitive: Boolean
```

Comment:
Indentation is inconsistent

Reply:
There was tab instead of space, updated.